c7n/schema.py

```diff
@@ -214,10 +214,19 @@ def generate(resource_types=()):
                 'name': {
                     'type': 'string',
                     'pattern': "^[A-z][A-z0-9]*(-[A-z0-9]+)*$"},
+
+                'conditions': {
+                    'type': 'array',
+                    'items': {'oneOf': [
```

Comment:
Does this include the `or`, `and`, and `not` types too to group things?  I'm not entirely clear on the schema so I can't tell...

Reply:
it does

Reply:
er.. I should say it does now ;-)